Projectile.cs

```diff
@@ -27,26 +27,12 @@ public Vector2 Velocity
             get { return velocity; }
             set { velocity = value; }
         }
-        private Vector2 ScrSize;
-        public Vector2 ScreenSize
-        {
-            get { return ScrSize; }
-            set { ScrSize = ScreenSize; }
-        }
-        private int spritecolor;
-        public int SpriteColor
-        {
-            get { return spritecolor;}
-            set { spritecolor = SpriteColor; }
-        }
 
         //maybe force a purple in if ot doesnt randpmly generate?
         //***DUSTIN
-        public Projectile(CombatSpriteColors newColor, Directions newDirection, Level currentLevel)
+        public Projectile(Level currentLevel)

```

Comment:
the if and else statements you wrote will be based on the properties of currentLevel and produce newColor and newDirection, which will still be set as SpriteColor and Direction, respectively